src/crepe/manager/ComponentManager.cpp

```diff
@@ -53,7 +53,7 @@ GameObject ComponentManager::new_object(const string & name, const string & tag,
 		this->next_id++;
 	}
 
-	GameObject object{*this, this->next_id, name, tag, position, rotation, scale};
+	GameObject object{this->mediator, this->next_id, name, tag, position, rotation, scale};
```

Comment:
waarom wordt de mediator nu meegegeven aan ieder game object?

Reply:
BehaviorScript krijgt op deze manier toegang tot meer dan alleen de ComponentManager, wat nodig is voor de nieuwe script utilities. Helaas moet dit helemaal doorgevoerd worden tot GameObject, maar alleen de ComponentManager wordt intern gebruikt door GameObject zelf, en Mediator wordt verder alleen gebruikt door BehaviorScript.